### ISSUE TITLE
Baby h

### ISSUE THREAD

**Issue by khrm** [bearcove/feedback#57]
**Where's the problem at?**
Please link to the article

**No, seriously, where is it?**
Copy/paste the paragraph the problem is in, maybe highlight what's wrong?

**Describe the issue and a fix if you know it**
In plain English, explain why this is wrong, and how to fix it (if you know how).

**Comment by fasterthanlime** [comment]
You didn't fill in the issue template at all.

**Comment by khrm** [comment]
Sorry. This issue was mistakely created when cellphone was in my pocket and your blog was opened in my chrome app.

**Comment by fasterthanlime** [comment]
No worries!
